### ISSUE TITLE
Safari does not show focus outline on SVG group

### ISSUE THREAD

**Issue by srallen** [zooniverse/front-end-monorepo#1757]
**Package** lib-classifier

**Describe the bug**
We have focus outline styles for drawing marks, which all of them are wrapped by SVG group elements. This element gets the focus style applied when the mark gets focus. 

Found while reviewing other focus bug fixes: https://github.com/zooniverse/front-end-monorepo/pull/1738#pullrequestreview-464394724

**To Reproduce**
Check out any mark story:

https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/story/drawing-tools-point--complete

Tab to said mark or use dev tools to force the focus state on the mark's parent group element in Safari. Compare to FF or Chrome, Safari gets the styles, but the outline is not visible. The outline is visible in FF and Chrome.

**Expected behavior**
Focus outline should be visible
